pages.de/common/stun.md

```diff
@@ -0,0 +1,12 @@
+# stun
+
+> Classic STUN client.
```

Comment:
```suggestion
> Klassischer STUN-Client.
```

Reply:
Classic here refers to the first RFC standard. Should we translate this anyway?

Reply:
Oh, ok. Then we shouldn't translate it.